client/src/components/EditProfile/ProfilePhotoTab/ProfilePhotoTab.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { Typography, Box, Grid, Avatar, Button, InputLabel } from '@material-ui/core';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { useAuth } from '../../../context/useAuthContext';
+import uploadImage from '../../../helpers/APICalls/uploadImage';
+import deleteImage from '../../../helpers/APICalls/deleteImage';
+import useStyles from './useStyles';
+
+const ProfilePhoto = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const [displayPhoto, setDisplayPhoto] = useState<string>('');
+  const [isUploaded, setIsUploaded] = useState<boolean>(false);
+  const [isWrongFileType, setIsWrongFileType] = useState<boolean>(false);
+
+  const onPhotoUploadChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const file: File = (e.target.files as FileList)[0];
+    const fileURL = file ? URL.createObjectURL(file) : '';
+    const images = [];
+    if (file && (file.type === 'image/jpeg' || file.type === 'image/png' || file.type === 'image/jpg')) {
+      setIsUploaded(true);
+      setIsWrongFileType(false);
+      setDisplayPhoto(fileURL);
+      if (loggedInUser) {
+        images.push(file);
+        uploadImage(images, loggedInUser.profile).then((data) => {
+          if (data.error) console.error(data.error);
+        });
+      }
+    } else setIsWrongFileType(true);
+  };
+
+  const onDeletePhotoClick = (): void => {
+    setIsUploaded(false);
+    if (loggedInUser) {
+      deleteImage(loggedInUser.profile).then((data) => {
+        if (data.error) console.error(data.error);
+      });
+    }
+  };
+
+  return (
+    <Grid container alignItems="center" direction="column" className={classes.root}>
+      <Grid item xs={12} sm={8} md={7}>
+        <Box display="flex" alignItems="center" justifyContent="center">
+          <Box width="100%" maxWidth={450} alignSelf="center">
+            <Typography color="textPrimary" className={classes.title}>
+              Profile Photo
+            </Typography>
+            <Grid container alignItems="center" direction="column">
+              <Grid item xs={12} sm={8} md={7}>
+                <Box display="flex" alignItems="center" justifyContent="center">
+                  <Box width="100%" maxWidth={450} alignSelf="center">
+                    {!isUploaded ? (
+                      <Avatar className={classes.avatar}>
+                        <AccountCircleIcon className={classes.icon} />
+                      </Avatar>
+                    ) : (
+                      <Avatar className={classes.avatar}>
+                        <img src={displayPhoto} className={classes.photo} />
```

Comment:
It's a good practice to add the `alt` attribute to `img` tag as this can be helpful to screen readers.